packages/web-api/src/types/response/AdminConversationsSearchResponse.ts

```diff
@@ -47,14 +47,15 @@ export interface Conversation {
   is_pending_ext_shared?:         boolean;
   is_private?:                    boolean;
   last_activity_ts?:              number;
-  lists?:                         Canvas;
+  lists?:                         ListsClass;
   member_count?:                  number;
   name?:                          string;
   pending_connected_team_ids?:    string[];
+  properties?:                    Properties;
   purpose?:                       string;
 }
 
-export interface Canvas {
+export interface ListsClass {
```

Comment:
Since this is exported isn't this name change a breaking change? 🤔

Reply:
It technically is, but this happens all the time due to the type generation script, which uses `quicktype` under the hood. We only export the `*Response` interfaces from the main package (see [here](https://github.com/slackapi/node-slack-sdk/blob/main/packages/web-api/src/types/response/index.ts) and [here](https://github.com/slackapi/node-slack-sdk/blob/main/packages/web-api/src/index.ts#L34)), so developers would have to explicitly import from `@slack/web-api/src/types/response/SomeAPIResponse` to get at these sub-interfaces. Ideally these would not be exported - I agree with you - but that would take some more work on the type generation.

Reply:
It could be, but this is inevitable as long as we auto-generate it using the 3rd party tool. We generally recommend explicitly referring only on XXXResponse types.